Library/Formula/memo.rb

```diff
@@ -0,0 +1,17 @@
+class Memo < Formula
+  url "http://www.getmemo.org/memo-1.5.tar.gz"
+  homepage "http://www.getmemo.org"
+  sha1 "3e047b09e91d695f1767d1dd6d179732c07a5759"
+
+  def install
+    system "make", "PREFIX=#{prefix}"
+    system "make", "install"
+    bin.install "memo"
+    man1.install "memo.1"
```

Comment:
The call to `make install` doesn't actually respect Homebrew's installation system. 

The two `XXX.install` calls accomplish the same thing as the call to `make install`, but put things in the right place and with the right permissions.

Also, since the make variable `PREFIX` doesn't actually do anything outside of the `install` target, it's use is pointless in line 7.